.github/actions/upgrade-and-test/action.yml

```diff
@@ -0,0 +1,157 @@
+name: Upgrade and test cluster
+
+description: >-
+  Run a Helm upgrade using the specified values, wait for the cluster to
+  become ready and run Sonobuoy against it
+
+inputs:
+  name:
+    description: The name of the cluster
+    required: true
+  os-client-config-file:
+    description: The path of the OpenStack clouds file
+    required: true
```

Comment:
should this be required or default - same goes for the rest of the inputs in this file that have both

Reply:
I think required + default prevents the input from being set to `null` in a caller.

Reply:
ah, haven't seen that before. Good to know!